src/material/form-field/testing/form-field-harness.ts

```diff
@@ -113,19 +113,20 @@ export class MatFormFieldHarness extends ComponentHarness {
     return labelEl ? labelEl.text() : null;
   }
 
-  /** Whether the form-field has a floating label. */
-  async hasFloatingLabel(): Promise<boolean> {
-    return (await this.host()).hasClass('mat-form-field-can-float');
-  }
-
```

Comment:
I'm proposing that we remove this method. It will only make sense in the non-MDC based form-field where `floatLabel = never` in the legacy appearance is supported. Not even adding it as part of the first stable release, will avoid a breaking change.

Reply:
It's not used in Google presently